compass/ocean/tests/utility/combine_topo/__init__.py

```diff
@@ -14,37 +19,54 @@ class CombineTopo(TestCase):
     datasets
     """
 
-    def __init__(self, test_group):
+    def __init__(self, test_group, target_grid):
         """
         Create the test case
 
         Parameters
         ----------
         test_group : compass.ocean.tests.utility.Utility
             The test group that this test case belongs to
+        target_grid : `str`, either "lat_lon" or "cubed_sphere"
         """
-        super().__init__(test_group=test_group, name='combine_topo')
 
-        self.add_step(Combine(test_case=self))
+        subdir = os.path.join('combine_topo', target_grid)
+
+        super().__init__(
+            test_group=test_group, name='combine_topo', subdir=subdir,
+        )
+
+        self.add_step(Combine(test_case=self, target_grid=target_grid))
 
 
 class Combine(Step):
     """
     A step for combining GEBCO 2023 with BedMachineAntarctica topography
     datasets
+
+    Attributes
+    ----------
+    target_grid : `str`, either "lat_lon" or "cubed_sphere"
+    resolution : `float` degrees, or `int` NExxx
+    resolution_name: `str`, either x.xxxx_degrees or NExxx
```

Comment:
To parse properly in the docs, these need to be formatted as:
```suggestion
    target_grid : str
        either "lat_lon" or "cubed_sphere"
    resolution : float or int
        degrees (float) or face subdivisions (int): NExxx
    resolution_name: str
        either x.xxxx_degrees or NExxx
```
Similarly for other docstrings below.

Reply:
See 7c98497d53171b8c17587e14090640ca62e564e6